core/include/webview/detail/backends/win32_edge.hh

```diff
@@ -778,8 +793,8 @@ private:
                         "put_IsStatusBarEnabled failed"};
     }
     add_init_script("function(message) {\n\
```

Comment:
From here bind is indirectly executed in the class initialisation flow. This may have been spawning reciprocal executions of the set_default_size lamda, but this should now be reduced to one.